homu/main.py

```diff
@@ -1027,6 +1027,12 @@ def create_merge(state, repo_cfg, branch, logger, git_cfg,
                     ok = False
             if state.squash:
                 try:
+                    commiter_name = subprocess.check_output(
+                        git_cmd('log', '-1', '--format="%an"',
+                                state.head_sha)).decode('ascii').strip()
+                    commiter_email = subprocess.check_output(
+                        git_cmd('log', '-1', '--format="%ae"',
+                                state.head_sha)).decode('ascii').strip()
```

Comment:
AFAICT, these are actually getting the *author's* info, not the committer's, so should the variables be renamed?

Reply:
Yes, you're right